modules/device/main.tf

```
@@ -71,6 +71,14 @@ resource "aws_ssm_parameter" "nerves_hub_device_ssm_secret_db_url" {
   tags      = var.tags
 }
 
+resource "aws_ssm_parameter" "nerves_hub_device_ssm_secret_db_pool_size" {
+  name      = "/${local.device_app_name}/${terraform.workspace}/POOL_SIZE"
```

```suggestion
  name      = "/${local.device_app_name}/${terraform.workspace}/DB_POOL_SIZE"
```
also where is this value used?  Does it get pulled automatically? 

and side note: good job not doing the dumb thing I did and try and add a KMS key to a non-secret 😉

---

https://github.com/smartrent/nerves_hub_web/blob/4863a6ee6a58ec9cdf979d564cdfe2c4b1fcb794/lib/nerves_hub/config.ex#L9

This needs to be POOL_SIZE